spiketoolkit/comparison/sortingcomparison.py

```diff
@@ -23,9 +36,11 @@ def __init__(self, sorting1, sorting2, sorting1_name=None, sorting2_name=None, d
             self._do_counting(verbose=verbose)
 
     def getSorting1(self):
+        # Samuel EDIT : why not a direct attribute acees  with self.sorting1 ?
```

Comment:
yeas I guess we could allow to access sorting1 ad sorting2 directly

Reply:
So are you OK for _sorting1 to sorting1 and remove this ?